runner/config.ts

```diff
@@ -128,6 +144,82 @@ export function getPackageName(options: Config): string|undefined {
   return basename;
 }
 
+/**
+ * Truncates the path to the slash after the last occurrence of the given
+ * package name.
+ *
+ * @param directory Name of directory
+ * @param path Path to be truncated
+ */
+export function truncatePathToDir(directory: string, path: string): string|
```

Comment:
we end up reimplementing these sorts of path manipulation helpers in so many of our different libraries. It's at the point where we should start looking into pulling them into a shared helper library.

Reply:
Any idea on a good place to do that?

Reply:
Oh, for a monorepo... polymer-analyzer, tools-common and polymer-project-config are the only really common packages we have right now. None of them seem great.